package.json

```diff
@@ -24,6 +25,7 @@
   "bugs": "https://github.com/pattern-lab/edition-node-gulp/issues",
   "author": "Brian Muenzenmeyer",
   "scripts": {
+    "start": "node_modules/gulp/bin/gulp.js patternlab:serve",
```

Comment:
I just learned that NPM creates aliases for all project-local executables in node_modules/.bin, and puts that in the the PATH environment variable when you run NPM scripts. Therefore, I think we don't need the full path to the local executable, which is what I think is breaking this on Windows.